packages/react-vapor/src/components/select/SelectWithFilter.tsx

```diff
@@ -28,6 +28,7 @@ export interface ISelectWithFilterOwnProps {
     noItemsText?: string;
     filterButton?: IButtonProps;
     filter?: IFilterBoxOwnProps;
+    filterPlaceholder?: string;

```

Comment:
I think you can send the filterPlaceholder inside the `filter` props, so you won't need the new props